packages/expo-av/src/Audio.types.ts

```diff
@@ -2,7 +2,7 @@
 export type AudioMode = {
   /**
    * A boolean selecting if recording is enabled on iOS.
-   * > When this flag is set to `true`, playback may be routed to the phone receiver instead of to the speaker.
+   * > When this flag is set to `true`, playback may be routed to the phone earpiece instead of to the speaker. Set it back to 'false' after stopping recording to reenable playback through the speaker.
```

Comment:
```suggestion
   * > When this flag is set to `true`, playback may be routed to the phone earpiece instead of to the speaker. Set it back to `false` after stopping recording to reenable playback through the speaker.
```

Formatting nit, it also would be nice if you can run `et gdad - p expo-audio` to regenerate the JSON data for the autogenerated docs.